auditbeat/_meta/fields.common.yml

```diff
@@ -50,6 +50,41 @@
         example: s0
         description: The object's SELinux level.
 
+    - name: process
+      type: group
+      description: >
+        These fields contain process and user information.
+        Available only on Linux when using the eBPF backend.
+
+      fields:
+      - name: entity_id
+        type: keyword
+        description: Globally unique identifier for a process.
+
+      - name: name
+        type: keyword
+        description: Process command.
+
+      - name: pid
+        type: integer
```

Comment:
I'm wondering if this should be long to match the ECS process.pid https://www.elastic.co/guide/en/ecs/current/ecs-process.html#field-process-pid

Reply:
Yes, I think you are correct